packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/EmptyStakingCard.tsx

```diff
@@ -69,86 +41,83 @@ export const EmptyStakingCard = () => {
         () => [
             {
                 id: 0,
-                icon: <Icon name="piggyBank" size={32} color={theme.iconPrimaryDefault} />,
+                icon: <Icon name="piggyBank" size="extraLarge" variant="primary" />,
                 title: <Translation id="TR_STAKE_ETH_SEE_MONEY_DANCE" />,
                 description: (
                     <Translation
                         id="TR_STAKE_ETH_SEE_MONEY_DANCE_DESC"
                         values={{
                             apyPercent: ethApy,
+                            t: text => (
+                                <Tooltip
+                                    dashed
+                                    isInline
+                                    content={<Translation id="TR_STAKE_APY_DESC" />}
+                                >
+                                    <abbr>{text}</abbr>
+                                </Tooltip>
+                            ),
```

Comment:
are you sure it's working correctly? Neither here, nor on the dashboard I can see the Tooltip :thinking:

Reply:
It's working. It's because it requires a change from Crowding that hasn't been synced yet.